mtp_api/apps/prison/models.py

```diff
@@ -7,5 +7,5 @@ class Prison(TimeStampedModel):
     nomis_id = models.CharField(max_length=3, primary_key=True)
     name = models.CharField(max_length=500)
 
-    def __unicode__(self):
+    def __str__(self):
```

Comment:
yep

On Thu, Jun 18, 2015 at 10:52 AM, Steve Marshall <notifications@github.com>
wrote:

> In mtp_api/apps/prison/models.py
> <https://github.com/ministryofjustice/money-to-prisoners-api/pull/4#discussion_r32714066>
> :
>
> > @@ -7,5 +7,5 @@ class Prison(TimeStampedModel):
> >      nomis_id = models.CharField(max_length=3, primary_key=True)
> >      name = models.CharField(max_length=500)
> >
> > -    def __unicode__(self):
> > +    def __str__(self):
>
> Oh, is it because Python 3?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/money-to-prisoners-api/pull/4/files#r32714066>
> .
>